### DISCUSSION TITLE
Add support for uploading files via camera

### DISCUSSION THREAD

**Discussion by curtisblackwell** [filamentphp/filament#10878] [Ideas]
[Filepond has a property called `captureMethod`](https://pqina.nl/filepond/docs/api/instance/properties/) that sets [the capture attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture).

It's a really simple way to allow uploading images with a camera on mobile. On desktop browsers, it behaves like a typical file field. I'm going to add support for this in my own app. Would a PR be accepted?

**Comment by emmanpbarrameda** [comment]
up for this feature

**Comment by danharrin** [comment]
Duplicate of #11974
